BraintreeCore/src/main/java/com/braintreepayments/api/DeviceInspector.kt

```diff
@@ -137,18 +117,5 @@ class DeviceInspector @VisibleForTesting internal constructor(
                     "$VENMO_APP_PACKAGE.$VENMO_APP_SWITCH_ACTIVITY"
                 )
             )
-
-        internal fun getDropInVersion(): String? {
```

Comment:
Will we have a way to track the drop in version once these changes go into drop-in? With this it looks like even in a drop-in integration the SDK would report the underlying core SDK version in analytics

Reply:
Great catch! On iOS, we aren't yet using iOS v6 (with FPTI) in Drop In - so the standard hasn't yet been set for this.

I can add this back, and then update our [tags doc](https://paypal.sharepoint.com/:x:/r/sites/BTSDK/_layouts/15/Doc.aspx?sourcedoc=%7B6F8E06EE-ECD0-47BD-9380-567AE4D3F2EF%7D&file=FPTI%20Tags%20-%20Mobile%20SDKs.xlsx&action=default&mobileredirect=true) to add `drop_in_sdk_version`. We _could_ re-use `c_sdk_ver` for the DropIn version, as well if you think that is more clear than having 2 separate?

Reply:
Eh - conflating the 2 values into one tag could get confusing

Reply:
Re-added & updated our tags doc!